docs/deploying.md

```diff
@@ -97,7 +97,7 @@ Resources:
   FckNatAsgLaunchConfig:
     Type: AWS::AutoScaling::LaunchConfiguration
     Properties:
-      ImageId: ami-05b6d5a2e26f13c93
+      ImageId: ami-0591f971d15aec0ab
       InstanceType: t4g.nano
```

Comment:
Let's remove the AMI ID update for now and just pull in the syntax fix. There's a broader problem here that the AMI ID doesn't account for the deployed region and I'd rather just pull it out into a parameter or put it in a map.

Reply:
Understood. Yeah i didn`t think about that before opening the PR. Reverted that change again.